themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -320,6 +320,15 @@ function openy_carnation_theme_suggestions_page_alter(array &$suggestions, array
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function openy_carnation_theme_suggestions_views_view_alter(array &$suggestions, array $variables) {
+  if($variables['view']->id() === 'latest_event_posts') {
+    $suggestions[] = 'views-view__latest_event_posts';
```

Comment:
I think, in this plave default view suggestions should work
views-view--latest_event_posts.html.twig

Reply:
Theme debug shows only views-view.html.twig suggestion there, so i added this one

Reply:
https://api.drupal.org/api/drupal/core%21modules%21views%21views.theme.inc/group/views_templates/8.2.x
Try read this and apply working suggestion